NOTICE

```diff
@@ -0,0 +1,9 @@
+Apache NiFi - MiNiFi
+Copyright 2014-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This product includes the following work from the Apache Hadoop project:
+
+BoundedByteArrayOutputStream.java adapted to SoftLimitBoundedByteArrayOutputStream.java
```

Comment:
Ah, good call on that one as we do not have the source bundled.  Will remove those lines